docs/reference/supported-environments.md

```diff
@@ -4,6 +4,15 @@ title: "Supported environments"
 description: "Find out where to run Camunda 8 components for SaaS and Self-Managed, including Optimize for both Camunda 8 and Camunda 7."
 ---
 
+:::note
+The versions listed on this page are the minimum version required if appended with a `+`.
+
+Pay attention to where the `+` falls, as most of our dependencies follow [semantic versioning](https://semver.org/) (semver), where `x.y.z` correspond to MAJOR.MINOR.PATCH. Higher or more recent versions will be compatible with Camunda, with respect to semver.
+
+For example, 1.2+ means support for the minor version 2, and any higher minors (1.3, 1.4, etc.) and patches (1.2.1, 1.2.2, etc.), but not majors, like 2.x.
```

Comment:
```suggestion
For example, 1.2+ means support for the minor version 1.2, and any higher minors (1.3, 1.4, etc.) and patches (1.2.1, 1.2.2, etc.), but not majors, like 2.x.
```
Shouldn't it be this?

Reply:
1 is the major version, 2 is the minor, but maybe this sentence could use some work.